kolibri_explore_plugin/assets/src/views/SearchPage.vue

```diff
@@ -283,6 +285,23 @@
         this.query = words.join(' ');
         this.$router.push({ params: { query: this.query } });
       },
+      onNodeUpdated(nodeId) {
+        ContentNodeResource.fetchModel({ id: nodeId }).then(newNode => {
```

Comment:
Isn't this going to suffer the same caching issue you were just addressing in #634?

Reply:
> Isn't this going to suffer the same caching issue you were just addressing in #634?

Nope. That is "the mistery of the day" to me. Why `ContentNodeResource.fetchModel` works well here and not in `kolibriApi`.